LICENSE

```diff
@@ -186,7 +188,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2016-2024 ClickHouse, Inc.
```

Comment:
ditto, should be kept as-is, you can refer to some ASF projects.

Reply:
same here as i have mention on the previous comment